lib/Installation/SystemRole/SystemRolePage.pm

```diff
@@ -25,9 +25,14 @@ sub new {
 }
 
 sub init {
-    my $self = shift;
+    my ($self) = @_;
     $self->SUPER::init();
     $self->{sel_role} = $self->{app}->itemselector({id => 'role_selector'});
+    $self->{btn_next} = $self->{app}->button({id => 'next'});
+    $self->{text_mode} = 'Text Mode';
```

Comment:
I was checking both screen and I got confused here with Text Mode,
https://openqa.suse.de/tests/7124541#step/system_role/1
https://openqa.opensuse.org/tests/1930998#step/system_role/6
They look completely different. I would say even more than before, so you use this Page as openSUSE page, could we represent all the roles here and for sle the same, all the ones listed in that screenshot, so in follow-up task they can be applied to all yamls.